Upstream-Status: Inappropriate

This patch file contains workarounds for GCC 6:

* device/u2f/u2f_ble_transaction.cc

  Use an explicit assignment to avoid the following error:

  ../../device/u2f/u2f_ble_transaction.cc: In member function 'void device::U2fBleTransaction::OnError()':
  ../../device/u2f/u2f_ble_transaction.cc:134:27: error: converting to 'std::queue<device::U2fBleFrameContinuationFragment, base::circular_deque<device::U2fBleFrameContinuationFragment> >' from initializer list would use explicit constructor 'std::queue<_Tp, _Sequence>::queue(_Sequence&&) [with _Tp = device::U2fBleFrameContinuationFragment; _Sequence = base::circular_deque<device::U2fBleFrameContinuationFragment>]'
     request_cont_fragments_ = {};
                                ^
--- a/device/u2f/u2f_ble_transaction.cc
+++ b/device/u2f/u2f_ble_transaction.cc
@@ -131,7 +131,7 @@ void U2fBleTransaction::StopTimeout() {
 
 void U2fBleTransaction::OnError() {
   request_frame_.reset();
-  request_cont_fragments_ = {};
+  request_cont_fragments_ = base::queue<U2fBleFrameContinuationFragment>();
   response_frame_assembler_.reset();
   std::move(callback_).Run(base::nullopt);
 }
